### ISSUE TITLE
docker-bosh-cli fails during v53.0.0 upgrade (not reproduced in v53.0.1)

### ISSUE THREAD

**Issue by JCL38-ORANGE** [orange-cloudfoundry/paas-templates#1490]
### Expected behavior
* As a paas-templates operator 
* In order to be confident with my platform upgrade
* I need all helmrelease deployed without errors

### Observed behavior
01-ci-k8s longhorn helm release fails 

### Traces in concourse
```
Task 7336408 | 09:42:32 | Updating instance docker-cli: docker-cli/b630441e-a358-4741-bb28-2c17c0724558 (0) (00:06:40)

                     L Error: 'docker-cli/b630441e-a358-4741-bb28-2c17c0724558 (0)' is not running after update. Review logs for failed jobs: datacontainer, xx, yy....   , ops_portal, route_registrar, bosh-dns, bosh-dns-resolvconf, cron, node_exporter, blackbox

Task 7336408 | 09:49:12 | Error: 'docker-cli/b630441e-a358-4741-bb28-2c17c0724558 (0)' is not running after update. Review logs for failed jobs: datacontainer, xx, yyy, , ops_portal, route_registrar, bosh-dns, bosh-dns-resolvconf, cron, node_exporter, blackbox
```

### Other traces
image not present 

docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
paas/ops_portal     53.0.0              66fef392ef8b        About an hour ago   8.77MB
alpine              3.12.0              a24bb4013296        20 months ago       5.57MB

/var/vcap/sys/log/docker/docker.stderr.log

docker-cli/b630441e-a358-4741-bb28-2c17c0724558:/var/vcap/sys/log/docker# tail -f docker.stderr.log
time="2022-01-27T10:27:06.203669849Z" level=info msg="Attempting next endpoint for pull after error: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 169.254.0.2:53: no such host"
time="2022-01-27T10:27:06.203834878Z" level=error msg="Handler for POST /v1.39/images/create returned error: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 169.254.0.2:53: no such host"
time="2022-01-27T10:27:35.334988430Z" level=info msg="Attempting next endpoint for pull after error: manifest unknown: The named manifest is not known to the registry."
time="2022-01-27T10:27:35.364249945Z" level=warning msg="Error getting v2 registry: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 169.254.0.2:53: no such host"
time="2022-01-27T10:27:35.364300215Z" level=info msg="Attempting next endpoint for pull after error: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 169.254.0.2:53: no such host"
time="2022-01-27T10:27:35.364336772Z" level=error msg="Handler for POST /v1.39/images/create returned error: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 169.254.0.2:53: no such host"
time="2022-01-27T10:28:05.500522944Z" level=info msg="Attempting next endpoint for pull after error: manifest unknown: The named manifest is not known to the registry."
time="2022-01-27T10:28:05.531652254Z" level=warning msg="Error getting v2 registry: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 169.254.0.2:53: no such host"
time="2022-01-27T10:28:05.531745485Z" level=info msg="Attempting next endpoint for pull after error: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 169.254.0.2:53: no such host"
time="2022-01-27T10:28:05.531832824Z" level=error msg="Handler for POST /v1.39/images/create returned error: Get https://registry-1.docker.io/v2/: dial tcp: lookup registry-1.docker.io on 169.254.0.2:53: no such host"

### Affected releases
v53.0.0

**Comment by JCL38-ORANGE** [comment]
works after monit stop/start atouil

**Comment by poblin-orange** [comment]
should be solved by migrating docker-bosh-cli to k8s #1268

**Comment by JCL38-ORANGE** [comment]
closed because docker boshcli was migrated in v55.
